### ISSUE TITLE
Are layers and artboards truly asset-universal?

### ISSUE THREAD

**Issue by trieloff** [adobe/xdm#11]
It seems to me that layers and artboards do not apply to all kinds of assets, only to certain images and certain documents.

**Comment by lrosenthol** [comment]
Correct - these are both specific to certain formats.  So they can span multiple formats, but certainly not all.  Layers are more generic than artboards.  I would remove that from the common/asset and move to an adobe namespace

**Comment by trieloff** [comment]
We can introduce new abstract schemas for `layerable` and `artbortable` (and come up with better names) and import them into new asset classes we create afterwards. 

The Adobe-specific file formats like INDD, AI, etc. should be in an Adobe namespace, but the class of assets that are:

* multi-paged
* have layers
* have artboards
* are rectangular

Are not specific to Adobe, so they can stay in XDM Core.

**Comment by lrosenthol** [comment]
@trieloff Yes, I think using the new "import" mechanism will be important here...and I think you have described such things.

**Comment by trieloff** [comment]
If you give me the names, I can refactor `layerable` and `artboartable`
